cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.1 on 2024-12-16 15:03
+
+import cinema.models

```

Comment:
Ensure that the `movie_image_path` function is defined in the `cinema.models` module. If it's not defined, this will cause an error when running the migration.